<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1000</width>
				<height>800</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Mass Slide Editor</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<layout class="QVBoxLayout" name="root_vertical_layout_for_banner_and_splitter">
				<!-- Renamed for clarity -->
				<item>
					<layout class="QHBoxLayout" name="top_banner_layout">						
						<item>
							<widget class="QLabel" name="section_title_banner_label">
								<property name="sizePolicy">
									<sizepolicy hsizetype="Expanding" vsizetype="Fixed"> <!-- Make label expand -->
										<horstretch>1</horstretch> <!-- Give it stretch priority -->
										<verstretch>0</verstretch>
									</sizepolicy>
								</property>
								<property name="text">
									<string>Section Title Banner</string>
								</property>
								<property name="alignment">
									<set>Qt::AlignCenter</set>
								</property>
								<property name="styleSheet"><string notr="true">font-size: 16pt; font-weight: bold; padding: 5px; border-bottom: 1px solid #aaaaaa;</string></property>
							</widget>
						</item>
					</layout>
				</item>
				<!-- New Fold-up Section using QGroupBox -->
				<item>
					<widget class="QGroupBox" name="section_properties_group_box">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Preferred" vsizetype="Maximum">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="title">
							<string>Section Properties</string>
						</property>
						<property name="checkable">
							<bool>true</bool>
						</property>
						<property name="checked">
							<bool>false</bool> <!-- Start collapsed -->
						</property>
						<layout class="QVBoxLayout" name="group_box_main_layout">
							<item>
								<widget class="QWidget" name="section_properties_content_container">
									<layout class="QVBoxLayout" name="section_properties_content_layout">
										<item>
											<!-- This QVBoxLayout will hold dynamically added metadata rows -->
											<layout class="QVBoxLayout" name="metadata_entries_layout">
												<!-- Initially empty, to be populated by Python code -->
												<!-- Example of how a row might look (for visualization, not to be static in UI):
												<layout class="QHBoxLayout">
													<item><widget class="QComboBox"/></item>
													<item><widget class="QLineEdit"/></item>
													<item><widget class="QPushButton" name="remove_metadata_button_example">
														<property name="text"><string>-</string></property>
														<property name="maximumSize"><size><width>30</width><height>16777215</height></size></property>
													</widget></item>
												</layout>
												-->
											</layout>
										</item>
										<item>
											<widget class="QPushButton" name="add_metadata_button">
												<property name="text">
													<string>Add Metadata Field</string>
												</property>
											</widget>
										</item>
										<item>
											<spacer name="metadata_vertical_spacer">
												<property name="orientation">
													<enum>Qt::Vertical</enum>
												</property>
												<property name="sizeHint" stdset="0">
													<size><width>20</width><height>40</height></size>
												</property>
												<property name="sizeType"><enum>QSizePolicy::Expanding</enum></property>
											</spacer>
										</item>
									</layout>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QSplitter" name="main_splitter">
						<property name="orientation">
							<enum>Qt::Horizontal</enum>
						</property>
						<widget class="QWidget" name="left_panel_widget">
							<!-- New container for main editor area -->
							<layout class="QVBoxLayout" name="main_editor_area_layout">
								<item>
									<widget class="QScrollArea" name="main_slides_scroll_area">
										<property name="widgetResizable">
											<bool>true</bool>
										</property>
										<widget class="QWidget" name="scrollAreaContentWidget">
											<property name="geometry">
												<rect>
													<x>0</x>
													<y>0</y>
													<width>698</width>
													<!-- This will adjust dynamically -->
													<height>776</height>
													<!-- This will adjust dynamically -->
												</rect>
											</property>
											<layout class="QVBoxLayout" name="slides_container_layout"></layout>
										</widget>
									</widget>
								</item>
							</layout>
						</widget>
						<widget class="QFrame" name="right_sidebar_frame">
							<property name="minimumSize">
								<size>
									<width>150</width>
									<height>0</height>
								</size>
							</property>
							<!-- Removed maximumSize to allow it to be resized larger -->
							<!-- <property name="maximumSize"><size><width>200</width><height>16777215</height></size></property> -->
							<property name="frameShape">
								<enum>QFrame::StyledPanel</enum>
							</property>
							<property name="frameShadow">
								<enum>QFrame::Raised</enum>
							</property>
							<layout class="QVBoxLayout" name="right_sidebar_layout">
								<item>
									<widget class="QPushButton" name="save_sidebar_button">
										<property name="text">
											<string>Save Changes</string>
										</property>
									</widget>
								</item>
								<item>
									<widget class="QLabel" name="slides_label">
										<property name="text">
											<string>Slides</string>
										</property>
										<property name="alignment">
											<set>Qt::AlignCenter</set>
										</property>
									</widget>
								</item>
								<item>
									<widget class="QScrollArea" name="right_slides_scroll_area">
										<property name="widgetResizable">
											<bool>true</bool>
										</property>
										<widget class="QListWidget" name="slide_thumbnails_list_widget">
											<property name="geometry">
												<rect>
													<x>0</x>
													<y>0</y>
													<width>178</width>
													<!-- This will adjust dynamically -->
													<height>746</height>
													<!-- This will adjust dynamically -->
												</rect>
											</property>
											<property name="frameShape">
												<enum>QFrame::NoFrame</enum>
											</property>
											<property name="frameShadow">
												<enum>QFrame::Plain</enum>
											</property>
										</widget>
									</widget>
								</item>
							</layout>
						</widget>
					</widget>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
	</ui>